R/lasnormalize.r

```diff
@@ -62,7 +62,7 @@
 #' \link[raster:raster]{raster}
 #' \link[lidR:grid_terrain]{grid_terrain}
 #' @export
-lasnormalize = function(.las, dtm, na.rm=...)
```

Comment:
What is that `na.rm = ...` ? I wrote that ? Impossible it is no a valid R code... :confused: